src/components/Table.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Dont commit this file. Its incomplete.

Reply:
noted

Reply:
Then remove it. Why is it still in the MR ?

Reply:
Delete this file and push the commit